### PR TITLE
Tests edit geom

### PR DESCRIPTION
**Pull request by SPlanzer**
Fixes: 
Predominately addresses epic [#139](https://github.com/linz/ds-group-epics/issues/139)

But also address editing functionality where porting missed API breakages 
* #167
* #168
* #168

### Notes for Testing:
Let automated tests run

#### Source Code Documentation Tasks:
- [ ] README updated (where applicable)
- [ ] CHANGELOG updated

#### User Documentation Tasks:
- [ ] Confluence user guide updated (where applicable)

#### Testing Tasks:
- [ ] Added tests that fail without this change
- [ ] All tests are passing in development environment
- [ ] Reviewers assigned

#### Pull Request Management:
- [ ] Linked to sub-task
- [ ] Linked to the epic

### CODE REVIEW

File: src/NZGBplugin/Layers.py [hunk 1]
```diff
@@ -291,7 +299,7 @@ def createLayers(self):
             self.moveLayersIntoGroup("search", "Gazetteer search results")
             self.moveLayersIntoGroup("feature", "Gazetteer feature")
 
-        # Find the group
+        # Find the groupMultiCurveM
```
Comment (billgeo) [inline src/NZGBplugin/Layers.py:302]: Typo here?

File: src/NZGBplugin/Layers.py [hunk 2]
```diff
@@ -12,6 +12,7 @@
 
 from builtins import str
 import os.path
+import sys
```
Comment (billgeo) [inline src/NZGBplugin/Layers.py:15]: Looks like this isn't used anywhere?

File: src/NZGBplugin/Layers.py [hunk 3]
```diff
@@ -609,9 +621,9 @@ def deleteSelectedGeometries(self):
             return
 
         gtypes = (
-            (QGis.Point, "point", "fpoint"),
-            (QGis.Line, "line", "fline"),
-            (QGis.Polygon, "polygon", "fpoly"),
+            (QgsWkbTypes.Point, "point", "fpoint"),
```
Comment (billgeo) [inline src/NZGBplugin/Layers.py:624]: Is this some other QGIS 2->3 API changes?

File: src/NZGBplugin/Layers.py [hunk 4]
```diff
@@ -523,13 +533,15 @@ def addSelectedGeometries(self):
...
             if geomlist:
                 if layer.crs() != self._dbCrs:
-                    ct = QgsCoordinateTransform(layer.crs(), self._dbCrs)
+                    ct = QgsCoordinateTransform(
+                        layer.crs(), self._dbCrs, QgsProject.instance()
+                    )
                     for g in geomlist:
                         g.transform(ct)
-                gtype = layer.geometryType()
+                gtype = layer.wkbType()
```
Comment (billgeo) [inline src/NZGBplugin/Layers.py:544]: 2->3 API change?

File: src/NZGBplugin/Layers.py (hunk 1 above)
Comment (SPlanzer) [reply src/NZGBplugin/Layers.py:302]: yip, fixed

File: src/NZGBplugin/Layers.py (hunk 2 above)
Comment (SPlanzer) [reply src/NZGBplugin/Layers.py:15]: removed

File: src/NZGBplugin/Layers.py (hunk 4 above)
Comment (SPlanzer) [reply src/NZGBplugin/Layers.py:544]: yip, see " General changes" https://qgis.org/api/api_break.html

File: src/NZGBplugin/Layers.py (hunk 3 above)
Comment (SPlanzer) [reply src/NZGBplugin/Layers.py:623]: All methods taking or returning QGis::WkbType have been changed to use QgsWkbTypes::Type 
(https://qgis.org/api/api_break.html)

**Pull request merged**